sql/expression/function/json_unsupported.go

```diff
@@ -56,6 +56,11 @@ func (j JSONContainsPath) FunctionName() string {
 	return "json_contains_path"
 }
 
+// Description implements sql.FunctionExpression
```

Comment:
All the functions in this file are unsupported, so should not be included in the docs.

I would add an `UnsupportedFunctionStub` interface extension and make all these implement it. Then check for it in the generation method.